### PR TITLE
Add POST /news entrypoint

### PR DESCRIPTION
**Pull request by Gabito-git**
ezgif-3-193239ea62

https://alkemy-labs.atlassian.net/browse/OT221-55

**Overviwew:**

- Added field validator middleware. It handles express-validator errors and responses with an array of errors.
- Added image file validator. It checks if a file is attached to the req.files object and if so, verifies the correct format. 
- Controller employs S3 service which is currently in code Review section.  It extracts the key value and save it to the DB in image field. This key value is used to retrieve the image from S3 bucket according to the developer in charge of that ticket. 
- The example was performed with a hardcoded string, but the pushed code handle the real upload to S3 bucket and its response.

### CODE REVIEW

File: src/middlewares/file-validator.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      ok: false,
+      err: 'No files were uploaded.'
+    });
+
+  }else {
+
+    // it extracts the filetype of the request.files.file object
+    const { mimetype } = req.files.file;
+    const fileType = mimetype.split('/')[0];
+    if(fileType !== 'imagre'){
```
Comment (GauntStrelok) [inline src/middlewares/file-validator.js:20]: esta bien escrito esto?  me parece que es image, y no imagre

**Review by GauntStrelok — CHANGES_REQUESTED** [review]
Fijate lo del if este que te marque, y podes cambiarlo, o podes contestarme si esta bien

**Comment by Gabito-git** [comment]
ezgif com-gif-maker

File: src/middlewares/file-validator.js (hunk 1 above)
Comment (Gabito-git) [reply src/middlewares/file-validator.js]: Si, mi error. Ya lo corregí y probé la corrección. En algún momento se me tecleó la "r" por error.

**Pull request merged**